web/portal/user/.hygen.js

```diff
@@ -0,0 +1,9 @@
+const helpers = require("@irontec/ivoz-ui/hygen/lib");
+
+module.exports = {
+  templates: `${__dirname}/node_modules/@irontec/ivoz-ui/hygen/templates`,
+  helpers: {
+    ...helpers,
+    url: () => "https://10.60.75.33/api/user/docs.json",
```

Comment:
There is a private ip address here. Is this file required? generated? Does it work with localhost?

Reply:
That file is only used for development purposes, anyway, there is no need to track such a file in git I will change .gitignore to avoid it.